try:
    1/0
except Exception as e:
    print("捕获到了异常")
